osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -92,6 +100,11 @@ public string SearchTerm
         /// </summary>
         protected virtual bool ShowPresets => false;
 
+        /// <summary>
+        /// Should overlay account for the multiplayer room global mods.
+        /// </summary>
+        public bool AccountForMultiplayerMods = false;
```

Comment:
I don't like where this is going. If logic for multiplayer must exist in a mod select overlay, then there should be a subclass of the overlay used by multiplayer.

Reply:
moved all added logic into separate class